test/symmetricDifference.js

```diff
@@ -44,4 +45,78 @@ describe('symmetricDifference', function() {
     eq(R.symmetricDifference(M, M2), []);
   });
 
+  // Remove duplicated values (if any) with respect to R.equals
```

Comment:
We have `uniq` which removes duplicate items from an Array. We should just use that if it works rather than creating a new function.

Reply:
Will do!

Reply:
Done